ansible_deployer/command_line.py

```diff
@@ -504,11 +504,9 @@ def main():
         workdir = None
 
     logger = set_logging(workdir, start_ts, options)
-
```

Comment:
I don't like commits just removing/adding blank lines it just makes git blame more dificult to read